github/scim_test.go

```diff
@@ -428,22 +429,31 @@ func TestUpdateAttributeForSCIMUserOptions_Marshal(t *testing.T) {
 	testJSONMarshal(t, u, want)
 }
 
-func TestListSCIMProvisionedIdentitiesOptions_Marshal(t *testing.T) {
-	testJSONMarshal(t, &ListSCIMProvisionedIdentitiesOptions{}, `{}`)
+func TestListSCIMProvisionedIdentitiesOptions_addOptions(t *testing.T) {
+	url := "some/path"
```

Comment:
Can we please keep this line:

```go
testJSONMarshal(t, &ListSCIMProvisionedIdentitiesOptions{}, `{}`)
```

for consistency?  We always test to see how an empty struct gets mashaled.

Reply:
Done! ✅

Reply:
Since the `ListSCIMProvisionedIdentitiesOptions` struct has no json tag (url only), the string/json fields should be the same as the structure fields, and no field should be omitted if empty.

I think the expected value should be something like this:

```go
testJSONMarshal(t, &ListSCIMProvisionedIdentitiesOptions{}, `{
	"StartIndex": null,
	"Count": null,
	"Filter": null
}`)
```
I just created a new issue so we can talk more about this subject. #2699